### ISSUE TITLE
Videos keeps buffering?

### ISSUE THREAD

**Issue by resting** [JadenGeller/Helium#34]
Anyone experiencing Youtube videos going into a infinite buffering mode after playing for like 10 secs or less?

**Comment by Aslan11** [comment]
Yes. I can't even get it to play a video, just stuck infinitely buffering. Video is loading snappy in chrome though? :confused:

**Comment by archshift** [comment]
This seems to be an issue with WebView in general, I experienced it while trying to use it for youtube videos in a different project.

**Comment by reidcooper** [comment]
I also get this, youtube video keeps forever buffering, while the browsers load the video flawlessly.

**Comment by jorenmartijn** [comment]
Same here, defeats the purpose of this app for me to be honest. :(

**Comment by JadenGeller** [comment]
It would be interesting to know what software/hardware you all are running. I've been unable to reproduce the issue.

**Comment by jorenmartijn** [comment]
@JadenGeller I've updated my post with my hardware and software info. Hope you'll be able to find a solution.

**Comment by lologhi** [comment]
Same problem here, on Yosemite 10.10.3 Macbook Pro 13'' 2015. (thx for the app !)

**Comment by martihag** [comment]
I had the same issue, and it seems to be related to the HTML5 player. After installing Flash, the videos play perfectly without buffer-freeze.

**Comment by resting** [comment]
I'm on OS X 10.10.3 too. iMac 27" late 2012.
Just downloaded Helium 1.6 and the video still gets stuck after like 10 secs.

@martihag How do you force it to play with Flash? I have Flash installed by Youtube always plays using HTML5.

**Comment by martihag** [comment]
@resting I dont think I did anything in particular. I did install Flash from Homebrew though.
```bash
$ brew cask install flash
```
And after that it seemed to prefer Flash over HTML5 in Helium. Still HTML5 in Safari.

**Comment by jorenmartijn** [comment]
@martihag Tried that, by the way, `cask` doesn't work but `brew install Caskroom/cask/flash` does. It doesn't seem to detect flash at all. Tried `brew install Caskroom/cask/flash-player` for good measure as well and still no dice.

**Comment by Siyfion** [comment]
@martihag That seems to have done the trick for me; using OSX Yosemite on a MacBook Pro Retina 2014.

**Comment by resting** [comment]
`brew install Caskroom/cask/flash` worked on my Macbook Air, but not on the iMac. Why? It was able to play YT videos in flash in my Macbook but not the iMac.

**Comment by wfalkwallace** [comment]
it seems to continue to work if you adjust the time-scrubber or quality (even if you set it to the same value). It seems like it's just not reacting properly if it reaches the end of its buffered content (if it never does this, it seems to work fine). Is there any way to retrigger a "start buffering" or something?

**Comment by dizballanze** [comment]
+1 macbook pro 15'' retina 2014, Yosemite

**Comment by MikaAK** [comment]
+1 macbook pro

**Comment by vincentorback** [comment]
+1 macbook pro

**Comment by ivobenedito** [comment]
+1 macbook pro. I've installed flash with brew-cask and it worked. 
Helium started using flash player from youtube and it doesn't freeze anymore.

**Comment by sharpnk** [comment]
Not being able to use HTML5 is actually a WebView limitation, it's not specific to Helium. I think this and #79 can both be fixed by #54.
